experimental/dds/tree2/src/core/tree/deltaUtil.ts

```diff
@@ -3,10 +3,67 @@
  * Licensed under the MIT License.
  */
 
-import { Mark, Root, Skip } from "./delta";
+import { Mutable } from "../../util";
+import { ITreeCursorSynchronous } from "./cursor";
+import { Root, DetachedNodeId, FieldChanges } from "./delta";
+import { rootFieldKey } from "./types";
 
 export const emptyDelta: Root<never> = new Map();
 
-export function isSkipMark(mark: Mark<unknown>): mark is Skip {
-	return typeof mark === "number";
+export const emptyFieldChanges: FieldChanges<never> = {};
```

Comment:
Not related to this PR, but we should probably do something like freezing this or changing the types to make it less likely that this is ever accidentally mutated. If this is only used in tests, we should probably move it to a test file to make it less likely to cause bugs in production.

Reply:
The types do prevent changes already for both of these.